src/app/games/tafl/tests/TaflMove.spec.ts

```diff
@@ -9,24 +9,30 @@ describe('TaflMove', () => {
 
     it('TaflMove creation, as a MoveCoordToCoord, should throw when created static', () => {
         const error: string = RulesFailure.MOVE_CANNOT_BE_STATIC();
+        // TODO: c'est quelque chose qu'on veut pas, un from qui throw! Ce test teste du code de test mal implémenté, est-il réellement utile ?!
         expect(() => MyTaflMove.from(new Coord(0, 0), new Coord(0, 0)))
             .toThrowError(error);
     });
-    it('should throw when given out of range coords', () => {
-        const outOfRange: Coord = new Coord(-1, -1);
-        const inRange: Coord = new Coord(0, 0);
-        expect(() => MyTaflMove.from(outOfRange, inRange))
-            .toThrowError('Starting coord of TaflMove must be on the board, not at (-1, -1).');
-        expect(() => MyTaflMove.from(inRange, outOfRange))
-            .toThrowError('Landing coord of TaflMove must be on the board, not at (-1, -1).');
-    });
-    it('TaflMove must throw move instruction message when diagonal', () => {
-        const error: string = TaflFailure.MOVE_MUST_BE_ORTHOGONAL();
-        expect(() => MyTaflMove.from(new Coord(0, 0), new Coord(1, 1))).toThrowError(error);
-    });
-    it('TaflMove must throw move instruction message when non linear', () => {
-        const error: string = TaflFailure.MOVE_MUST_BE_ORTHOGONAL();
-        expect(() => MyTaflMove.from(new Coord(0, 0), new Coord(2, 5))).toThrowError(error);
+
+    describe('isValidStandAndEnd', () => {
```

Comment:
rajouter en plus le test que quand ça foire, ça throwAndLog ?

Reply:
il y a un TODO quelques lignes plus haut que t'as raté !
Ici, isValidStartAndEnd ne throw pas, that's the point!

Reply:
Après des coord out of board, ça doit throw quand ça arrive, si il est impossible de faire ça avec l'interface, non ?

Reply:
ah mais oui j'suis épais, fallait pas rendre des throw en failure car ceux-ci n'étaient pas reachable !